concrete_datastore/settings/base.py

```diff
@@ -220,8 +220,25 @@
         'rest_framework.renderers.JSONRenderer',
         'rest_framework.renderers.BrowsableAPIRenderer',
     ),
+    'DEFAULT_THROTTLE_CLASSES': (
+        'concrete_datastore.api.v1.throttling.CustomAnonymousRateThrottle',
+        'concrete_datastore.api.v1.throttling.CustomUserRateThrottle',
+    ),
 }
 
+#: Throttling rate should be (requests / duration)
+#: Accepted durations are all the strings that start with
+#: - "s" (seconds) example: s, sec, second, seconds, ...
+#: - "m" (minutes) example: m, min, minute, minutes, ...
+#: - "h" (hours) example: h, hou, hour, hours, ...
+#: - "d" (days) example: d, day, days, ...
+#: Please refer to the implementation of the method for more info
+#: https://github.com/encode/django-rest-framework/blob/3.10.2/rest_framework/throttling.py#L106
+ENABLE_AUTHENTICATED_USER_THROTTLING = True
+USER_THROTTLING_RATE = '500/m'
+ENABLE_ANONYMOUS_USER_THROTTLING = True
+ANONYMOUS_THROTTLING_RATE = '500/m'
```

Comment:
By default, change `ANONYMOUS_THROTTLING_RATE` to `6/m`

Reply:
changed